dlna/dms/dms_unix_test.go

```diff
@@ -1,21 +1,21 @@
-//+build unix
+//+build linux darwin
 
 package dms
 
 import "testing"
 
 func TestIsHiddenPath(t *testing.T) {
 	var data = map[string]bool{
-		"/some/path": false,
-		"/some/foo.bar": false,
+		"/some/path":         false,
+		"/some/foo.bar":      false,
 		"/some/path/.hidden": true,
 		"/some/.hidden/path": true,
-		"/.hidden/path": true,
+		"/.hidden/path":      true,
 	}
 	for path, expected := range data {
 		if actual, err := isHiddenPath(path); err != nil {
 			t.Errorf("isHiddenPath(%v) returned unexpected error: %s", path, err)
-		] else if expected != actual {
+		} else if expected != actual {
```

Comment:
A typo showing that this file was never used :eyes:!

Reply:
😮